Proposals/xgov-15.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Algorand Inside
+author: Trendheo (@hungstock1)
+category: Community
+focus_area: Social
+open_source: Yes
+amount_requested: 9000
+status: Draft
```

Comment:
Remember to change this to "Final" if you want your proposal to go through the next voting session.

Reply:
i fixed! thanks!